scripts/scil_tractogram_resample.py

```diff
@@ -56,29 +63,33 @@ def _build_arg_parser():
 
     p.add_argument('--never_upsample', action='store_true',
                    help='Make sure to never upsample a tractogram.\n'
-                        'Useful when downsample batch of files using bash.')
+                        'Useful when downsampling a batch of files using '
+                        'bash.')
```

Comment:
I would remove "using bash". Kind of implicit by mentionning a "batch of files" and not everyone uses bash (Mac uses zsh by default, for example)